### DISCUSSION TITLE
Controlling two recorders serially

### DISCUSSION THREAD

**Discussion by wehimwich** [nvi-inc/fs#159] [Ideas]
In [#153](https://github.com/nvi-inc/fs/discussions/156#discussioncomment-1595232), @vlbi-jun-yang wrote:

> At the end of this year, each EVN station is supposed to get a second Flexbuff to double the available disk space. While, I am afraid that the goal might not be reached for various reasons at some stations.
> 
> These Flexbuffs are supposed to be used serially for the normal VLBI observations with DBBC2. It is better to use a script to change the recorder. This script will help us to edit the control file mk5ad.ctl, run the command "fila10g=destination ..." and then restart FS.

**Comment by wehimwich** [comment]
With a fairly modest change (to allow rerunning _mk5cn_ in initialization mode), it should be possible to trigger changing to a different recorder while the FS is running. Of course this should not be done while the current recorder is recording. This mechanism can be used by the script you mention.

Would it be useful to have this triggered automatically by the schedule? If so, we could consider ways of doing that.

**Comment by vlbi-jun-yang** [comment]
If FS provides a new feature to re-initialize _mk5cn_ without restarting FS, it would be great. This may be also served as an extension of _mk5relink_. If we need to change the recorder, I will put the script in the procedure _sched_initi_, which has been re-defined to initilize some local equipments in the experiement prc file at Onsala.  So, this change will be automatically done.
